cms/templates/cms/home_page.html

```diff
@@ -10,122 +10,161 @@
-    </a>
-  </div>
-  <div class="header-links-dashboard">
-    <a href="/dashboard">Dashboard</a>
-  </div>
-  <div class="header-links-login">
-    {% if authenticated %}
-    <a class="btn btn-outline btn-success" href="/logout">
-      Sign out
-    </a>
-    {% else %}
-    <a class="btn btn-outline btn-success" href="{% url 'social:begin' 'edxorg' %}">
-      Sign in
-    </a>
-    {% endif %}
-  </div>
-</div>
+<div class="app-media layout-boxed">
 
-<div class="main-content-wrapper app-media">
-  <section class="main-content">
+  <nav class="navbar navbar-default" role="navigation">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="javascript:void(0)"><img src="/static/images/logo-micromasters@2x.png" width="215" alt="MIT Micromasters" /></a>
+        <div class="nav-utility pull-right">
+          {% if authenticated %}
+          <a class="btn btn-danger" href="/logout">
+            Sign out
+          </a>
+          {% else %}
+          <span>Get Started Today!</span>
+          <a class="btn btn-danger" href="{% url 'social:begin' 'edxorg' %}">
+            Sign in with edX.org
+          </a>
+          {% endif %}
+        </div>
+        {% if authenticated %}
+        <ul class="nav navbar-toolbar navbar-left">
+          <li class="active">
+            <a href="/dashboard">Dashboard</a>
+          </li>
+        </ul>
+        {% endif %}
+      </div>
+    </div>
+  </nav>
 
+  <div class="main-content-wrapper">
+    <section class="main-content">
 
-    <div class="banner-wrapper">
-      <div class="banner-wrapper-content">
-        <h1>
-          {{ title }}
-        </h1>
-        <a href="#" role="button" class="btn btn-primary btn-lg btn-learn-more">
-          <span>LEARN MORE</span>
-        </a>
-      </div>
-      <div class="banner-bottom">
+      <div class="banner-wrapper">
         <div class="banner-wrapper-content">
-          <h2>Get Started Today!
-            <a href="{% url 'social:begin' 'edxorg' %}" role="button" class="btn btn-success sign-in-button margin-left-10">
-              <span>Sign in with edX.org</span>
-            </a>
-          </h2>
+          <h1>
+            <div class="banner-image">
+              <img src="static/images/logo-mit-verylg@2x.png" alt="MIT" />
+            </div>
+            {{ title }}
+          </h1>
         </div>
       </div>
-    </div>
 
+      <div class="row sub-banner">
+        <div class="col-md-12">
+          <div class="sub-banner-content">
+            MIT has created a new credential for online learners: the Micromasters, which will be granted by <a href="https://www.edx.org/school/mitx" target="_blank">MITx</a> to students who do exceptionally well in a given set of graduate-level online courses and do well in a subsequent exam. <a href="javascript:void()" data-target="#modal-about" data-toggle="modal">Learn more</a>.
```

Comment:
Is 'Micromasters' meant to be plural here? It is in the original template. cc @pdpinch